doc/rados/operations/add-or-rm-osds.rst

```diff
@@ -2,195 +2,214 @@
  Adding/Removing OSDs
 ======================
 
-When you have a cluster up and running, you may add OSDs or remove OSDs
-from the cluster at runtime.
+When a cluster is up and running, it is possible to add OSDs or remove OSDs. 
 
 Adding OSDs
 ===========
 
-When you want to expand a cluster, you may add an OSD at runtime. With Ceph, an
-OSD is generally one Ceph ``ceph-osd`` daemon for one storage drive within a
-host machine. If your host has multiple storage drives, you may map one
-``ceph-osd`` daemon for each drive.
+OSDs can be added to a cluster in order to expand the cluster. Typically, an
+OSD is a Ceph ``ceph-osd`` daemon running on one storage drive within a host
+machine. But if your host machine has multiple storage drives, you may map one
+``ceph-osd`` daemon for each drive on the machine.
 
-Generally, it's a good idea to check the capacity of your cluster to see if you
-are reaching the upper end of its capacity. As your cluster reaches its ``near
-full`` ratio, you should add one or more OSDs to expand your cluster's capacity.
+It's a good idea to check the capacity of your cluster to see whether it is
+approaching the limit of its capacity. If your cluster has reached its ``near
+full`` ratio, then you should add one or more OSDs to expand your cluster's
+capacity.
+
+.. warning:: Do not add an OSD after your cluster has reached its ``full
+   ratio``.  OSD failures that occur after the cluster reaches its ``near full
```

Comment:
@zdover23 - A non-puzzling introduction of double spaces (it's the product of a sentence beginning on a new line).

Reply:
I agree with @anthonyeleven. I'll remove double spaces where it's convenient, but they render the same as single spaces, so it's not the most pressing documentation issue in the Ceph project.